### PR TITLE
refactor: Rename binary and default directory

### PR DESCRIPTION
**Pull request by hacheigriega**
## Explanation of Changes

- Binary name has been renamed from `seda-chaind` to `sedad`
- Default directory name has been renamed from `.seda-chain` to `.sedad`

### CODE REVIEW

**Comment by mariocao** [comment]
> Approving, but one concern I have is if we are renaming the home directory the chain uses to `~/.seda` and if a person runs both a chain and an overlay node on the same PC they are now using the same directory. In this case, I would be concerned about logs overwriting each other from either binary and user UX, but I don't know if a person is likely to run both on the same machine. I don't see why not, considering the overlay is very light. Maybe we should consider `~/.seda/chain` on the chain side and `~/.seda/overlay` on the overlay side.

I don't mind changing the folder as long as we make sure that they is no overwriting. 😅

File: cmd/sedad/__debug_bin1406259301
[the remark this answers is not in the document]
Comment (jim380) [reply cmd/sedad/__debug_bin1406259301:1]: Agreed. It's common for node operators to do `rm -rf ~/.seda` which would remove the overlay data dir with it. 

There is actually a simpler approach, simply do `~/.sedad` like osmosis does (`~/.osmosisd`).

[the remark this answers is not in the document]
Comment (mariocao) [reply cmd/sedad/__debug_bin1406259301:1]: I like it! 👍

**Comment by Thomasvdam** [comment]
We should also go over our other projects/docs and update those. The contracts repo has some references that I know of and there's probably more.

**Pull request merged**